pinax/stripe/actions/charges.py

```diff
@@ -96,7 +96,11 @@ def sync_charge_from_stripe_data(data):
     Returns:
         a pinax.stripe.models.Charge object
     """
-    customer = models.Customer.objects.get(stripe_id=data["customer"])
+    customer_id = data["customer"]
+    if customer_id:
+        customer = models.Customer.objects.get(stripe_id=customer_id)
+    else:
+        customer = None
```

Comment:
I've just noticed that #307 takes a different approach to this.

The code is shorter, but does a lookup that might not be required.